Tao_Current.txt

```diff
@@ -1,73 +1,84 @@
 
 The IETF is always in a state of change. Although the principles in this document are expected to remain largely the same over time, practical details may well have changed by the time you read it; for example, a web-based tool may have replaced an email address for requesting some sort of action.
 
-Many types of IETF documentation are mentioned in the Tao, from BCPs to RFCs and STDs. BCPs make recommendations for Best Current Practices in the Internet; RFCs are the IETF's main technical documentation series, politely known as "Requests for Comments"; and STDs are RFCs identified as "standards". Actually, all three types of documents are RFCs; see Section 6 for more information.
+Many types of IETF documentation are mentioned in the Tao, from BCPs to RFCs and STDs. The IETF publishes its main technical documentation as RFC, politely known as "Requests for Comments"; and STDs are RFCs identified as "standards". BCPs make recommendations for Best Current Practices in the Internet and are also RFCs; All three types of documents are in the RFC document stream; see Section 6 for more information.
 
 This web page is a continuation of the series of the "Tao of the IETF" RFCs. See [RFC6722] for an explanation of how the last RFC in that series became this web page. This web-based version of the Tao is based on [RFC4677], was co-authored with Susan Harris. The original version of this document, published in 1994, was written by Gary Malkin.
 
 So, why "the Tao"? Pronounced "dow", Tao is the basic principle behind the teachings of Lao-tse, a Chinese master. Its familiar symbol is the black-and-white yin-yang circle. Taoism conceives the universe as a single organism, and human beings as interdependent parts of a cosmic whole. Tao is sometimes translated "the way", but according to Taoist philosophy the true meaning of the word cannot be expressed in words.
+
 1.1 Acronyms and Abbreviations Used in the Tao
 
 Some of the acronyms and abbreviations from this document are listed below.
-Term 	Meaning
-AD 	Area Director
-BCP 	Best Current Practice
-BOF 	Birds of a Feather
-FAQ 	Frequently Asked Question(s)
-FYI 	For Your Information (RFC)
-IAB 	Internet Architecture Board
-IAD 	IETF Administrative Director
-IANA 	Internet Assigned Numbers Authority
-IAOC 	IETF Administrative Oversight Committee
-IASA 	IETF Administrative Support Activity
-ICANN 	Internet Corporation for Assigned Names and Numbers
-I-D 	Internet-Draft
-IESG 	Internet Engineering Steering Group
-IETF 	Internet Engineering Task Force
-IPR 	Intellectual property rights
-IRTF 	Internet Research Task Force
-ISOC 	Internet Society
-RFC 	Request for Comments
-STD 	Standard (RFC)
-WG 	Working Group
+
+| Term   | Meaning                                             |
+|--------|-----------------------------------------------------|
+| AD     | Area Director                                       |
+| BCP    | Best Current Practice                               |
```

Comment:
Do you want to point out here that this is a kind of RFC.

Reply:
Accepted and changed